components/src/Settings/schema.js

```diff
@@ -5,15 +5,20 @@ export const defaultSettings = {
     decrementCount: "Alt+Shift+-",
     resetCount: "Alt+Shift+0"
   },
-  alwaysOnTop: false,
+  alwaysOnTop: true,
   counterMode: 'manual', /* manual, ptFileWatcher */
   ptFileWatcherMode: {
     mode: 'track', /* track, playlist */
     subMode: 'all', /* all, specific */
     offset: 0,
     trackName: '',
     audioFilesPath: ''
-  }
+  },
+  TakeDisplaySettings: {
```

Comment:
Just a minor formatting thing, could you please make the `T` in `TakeDisplaySettings` lower case to match the convention? Typically things will start with a lower case letter unless they're the name of a class (and in React the name of a component).